Automattic/newspack-blocks
# Extract class/function definitions from plugin entry file

### All Submissions:






### Changes proposed in this Pull Request:

This PR aims to separate where classes and functions are declared and where they are used. Previously, those two things were mixed in `newspack-blocks.php` and this PR splits the class into its own file (following a pattern that was established with the API class).

Apart from the class extraction, it also takes two global functions `newspack_blocks_image_size_for_orientation` and `newspack_blocks_image_sizes` and it puts them inside `Newspack_Class` to keep all global things under one umbrella.

### How to test the changes in this Pull Request:

1. Install this plugin in your WP environment and use as normally.
2. There should be zero changes in functionality as the code is the same, it has just changed location from one file to another.
3. Obviously, no errors being output.

### Other information:
